docs/_includes/src-highlightjs.adoc

```diff
@@ -0,0 +1,22 @@
+////
+Included in:
+
+- user-manual: Source Code Syntax Highlighting: HighlightJs installation
+
+:highlightjs-lang: https://highlightjs.org/download/
+////
+
+{highlightjs-org}[HighlightJS] is a popular client-side syntax highlighter that supports a broad range of {highlightjs-lang}[languages].
+
+To use highlightjs just add `:source-highlighter: highlightjs` in your AsciiDoc file.
+
+[IMPORTANT]
+====
+Asciidoctor defaults to the basic bundle shipped by cdnjs, v8.9.1.
```

Comment:
Let's not mention the specific version or else we have to constantly update this when core changes. Let's just say "tracks the most recent release hosted on cdnjs". Technically, we are one behind right now, but that's because there was a breaking change in highlight.js and we haven't yet tested it.